### DISCUSSION TITLE
SSL connection timeout
Why is the copy process failing? 
Please help as soon as possible

### DISCUSSION THREAD

**Discussion by Aishwow** [community/community#63310] [Codespaces] [answered]
SSL connection timeout

**Comment by nathan-hello** [comment, marked as answer]
Which SSL connection? The one to your codespace? Could you give any logs or screenshots? Your post is sparse on details so it's hard to help.

**Reply by Aishwow** [reply]
git clone https://github.com/msn-05/termux-fish.git
Cloning into 'termux-fish'...
fatal: unable to access 'https://github.com/msn-05/termux-fish.git/'
![IMG_٢٠٢٣٠٨٠٩_٢٢٥٨١٣](https://github.com/community/community/assets/138838501/6501c285-2417-40db-b463-30dc9424c40f)
: SSL connection timeout

**Reply by nathan-hello** [reply]
In your screenshot, you did `git clone https://github.com/derv82/wifite`. With git clone (at least on Github and everywhere else I've seen), it's important to have `.git` at the end of the url. So try `git clone https://github.com/derv82/wifite.git`

**Reply by MohammedDom** [reply]
Did you find the solution

**Reply by MohammedDom** [reply]
I have done everything and always get the same result as in the picture

**Reply by Aishwow** [reply]
I did it many times but it didn't work before I was using the tools from the repositories normally now I can't copy any link from github

**Reply by Aishwow** [reply]
No bro, I don't know what to do

**Comment by Aishwow** [comment]
> في لقطة الشاشة الخاصة بك ، فعلت `git clone https://github.com/derv82/wifite`. باستخدام git clone (على الأقل على Github وفي أي مكان آخر رأيته) ، من المهم أن يكون موجودًا `.git`في نهاية عنوان url. لذا حاول`git clone https://github.com/derv82/wifite.git`

**Comment by MohammedDom** [comment]
I've done everything and always get the same result

في الجمعة، 11 أغسطس 2023 4:42 ص hafid almuk ***@***.***> كتب:
